posthog/settings/sentry.py

```diff
@@ -16,8 +16,10 @@
         sentry_logging = sentry_logging = LoggingIntegration(level=logging.INFO, event_level=None)
         sentry_sdk.init(
             dsn=os.environ["SENTRY_DSN"],
+            environment=os.getenv("SENTRY_ENVIRONMENT", "production"),
             integrations=[DjangoIntegration(), CeleryIntegration(), RedisIntegration(), sentry_logging],
             request_bodies="always",
+            sample_rate=1.0,  # Configures the sample rate for error events, in the range of 0.0 to 1.0. The default is 1.0 which means that 100% of error events are sent. If set to 0.1 only 10% of error events will be sent. Events are picked randomly.
             send_default_pii=True,
-            environment=os.getenv("SENTRY_ENVIRONMENT", "production"),
+            traces_sample_rate=0.01,  # A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app.
```

Comment:
That's 1% - PR description mentioned 0.1% which would be 0.001
```suggestion
            traces_sample_rate=0.001,  # A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app.
```

Reply:
True brainfart #2 🤣 I thought I opened the PR yesterday mentioning 1%. I clearly need a coffee this morning.